gui/default/syncthing/folder/restoreVersionsModalView.html

```diff
@@ -1,4 +1,4 @@
-<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions' | translate}}" large="yes" closeable="yes">
+<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions :' | translate}} {{folderLabel (restoreVersions.folder)}}" large="yes" closeable="yes">
```

Comment:
Same as on the other, instead of `Restore Versions: somefolder` which requires re-translation of the `Restore Versions` string, I'd go with `Restore Versions (somefolder)` and have the folder name outside the translated string.